Fluid/MemberNameStrategies.cs

```diff
@@ -7,7 +7,7 @@ public class MemberNameStrategies
     {
         public static readonly MemberNameStrategy Default = RenameDefault;
         public static readonly MemberNameStrategy CamelCase = RenameCamelCase;
-        public static readonly MemberNameStrategy SnakeCase = RenameCamelCase;
+        public static readonly MemberNameStrategy SnakeCase = RenameSnake;
```

Comment:
Good catch, can your rename the method to `RenameSnakeCase`

Reply:
done~